JASP-Desktop/qquick/jasptheme.cpp

```diff
@@ -133,7 +138,12 @@ void JaspTheme::setCurrentTheme(JaspTheme * theme)
 	_currentTheme = theme;
 
 	if(_currentTheme)
+	{
 		emit _currentTheme->jaspThemeChanged(theme);
+		_currentTheme->updateFontMetrics();
```

Comment:
Last comment: I would inverse these 2 lines

Reply:
Yeah ok, that is reasonable